src/Form/ManifestSettingsForm.php

```diff
@@ -132,7 +132,7 @@ public function buildForm(array $form, FormStateInterface $form_state) {
       '#type' => 'managed_file',
       '#title' => $this->t('General App Icon'),
       '#description' => $this->t('Provide a square (.png) image. This image serves as your icon when the user adds the website to their home screen. <i>Minimum dimensions are 512px by 512px.</i>'),
-      '#default_value' => [$fid],
+      '#default_value' => $fid,
```

Comment:
I think we can just replace this by the following and remove the `$fid` variable altogether because it's not used anywhere else.

```suggestion
      '#default_value' => $icon ?: [],
```

The ternary operator should work here because we know that the `$icon` variable is set.